plutus-core/plutus-ir/src/PlutusIR/Transform/Inline.hs

```diff
@@ -117,16 +113,26 @@ turn into a delay/force pair and get simplified away, and then we have something
 inline. This is essentially the reason for the existence of the UPLC inlining pass.
 -}
 
--- 'SubstRng' in the paper, no 'Susp' case
+-- | Substitution range, 'SubstRng' in the paper.
```

Comment:
To be clear, I don't think that "substitution range" is a good name, which is why I didn't use it! It's helpful to identify links to the paper, we _don't have to follow it_, in terminology or otherwise.

Reply:
I was wondering why it wasn't used! I think it's nice to follow it though because we are following the paper for these things.